Package.swift

```diff
@@ -6,19 +6,20 @@ let package = Package(
     dependencies: [
         // Vapor
         .package(url: "https://github.com/vapor/fluent-mysql.git", from: "3.0.0"),
+        .package(url: "https://github.com/vapor/fluent-sqlite.git", from: "3.0.0"),
         .package(url: "https://github.com/vapor/redis.git", from: "3.0.0"),
         .package(url: "https://github.com/vapor/vapor.git", from: "3.0.0"),
 
         // Nodes
-        .package(url: "https://github.com/nodes-vapor/admin-panel.git", .upToNextMinor(from:"2.0.0-beta")),
-        .package(url: "https://github.com/nodes-vapor/bugsnag.git", .branch("vapor-3")),
-        .package(url: "https://github.com/nodes-vapor/jwt-keychain.git", from: "1.0.0-beta"),
+        .package(url: "https://github.com/nodes-vapor/admin-panel.git", .upToNextMinor(from:"2.0.0-rc")),
```

Comment:
Let's keep it consistent and use:
```suggestion
        .package(url: "https://github.com/nodes-vapor/admin-panel.git", from: "2.0.0-rc"),
```